index.js

```diff
@@ -286,3 +309,47 @@ function toBuffer (chunk, encoding) {
     ? Buffer.from(chunk, encoding)
     : chunk
 }
+
+/**
+ * Most browsers send "br" (brotli) as the last value in
+ * in the 'Accept-Encoding' header which causes it to be
+ * deprioritized according to the spec.
+ *
+ * This is typically not what end users actually want so here
+ * we force the "br" (brotli) value to first in the list so that
+ * it will get properly prioritized and used.
+ *
+ * It's worth noting that although this is not "spec compliant",
+ * we belive it follows a well-established convention.
+ *
+ * @private
+ */
+function prioritize (str) {
```

Comment:
Looking at this, it seems like this can be an option added to the accept negotiation logic directly? Like an option to prefer server order over client order? I think that is effectively what this is doing, or am I misunderstanding?

Reply:
Great idea. I have added a `priorizeClient` option that will allow users to opt-out of this functionality. 

For what it's worth, we were already using this behavior because we were prioritizing GZIP over deflate even when deflate came first. As such, I think "opt-out" vs "opt-in" is acceptable in this case.

Reply:
I'm sorry, I think I didn't explain very well my thought on this; it wasn't to add a parameter here to this library's API, but rather to the accepts API. As far as adding this as a parameter to this module's API, I'm not sure if it's actually necessary, though if it is going to be surfaced as a switch, I would kind of expect such a switch to default to the spec compliant method. But again, I'm not sure if this actually needs to be a switch for the user; a better user switch would actually be to turn on/off each individual compression method, but that would be out of scope for this PR.

Reply:
I see what you are saying but we were already out of spec because we were prioritizing `gzip` over `deflate` even when deflate came first. The option to be fully spec compliant would be a *new* feature for this library.

Reply:
I think you are miss understanding what I am saying on this comment again, I'm really sorry. I must not be good at communicating my idea here through writing and that is my bad. I tried twice and it wasn't successful :( Perhaps if you desiure to move this forward would you be willing to do some kind of voice call?

Reply:
@wesleytodd This was already included in version 2 of `accepts`, right?